LatyshevEV/lab_5/ConsoleApplication2.cpp

```diff
@@ -0,0 +1,142 @@
+﻿#include "pch.h"
+#include <algorithm>
+#include <iostream>
+#include <string>
+#include <conio.h>
+using namespace std;
+int const max_size = 10;
+void input(int* matrix, int length, int height)
+{
+	for (int i = 0; i < height; i++)
+	{
+		for (int k = 0; k < length; k++)
+		{
+			int a;
+			cin >> a;
+			*(matrix + i * max_size + k) = a;
+		}
+	}
+}
+void my_print(int* matrix, int length, int height)
+{
+	for (int i = 0; i < height; i++)
+	{
+		for (int k = 0; k < length; k++)
+		{
+			cout << *(matrix + i * max_size + k) << ' ';
+		}
+		cout << endl;
+	}
+}
+
+void summ(int* matrix1, int* matrix2, int* result_matrix, int length1, int length2, int height1, int height2)
+{
+		for (int i = 0; i < length1; i++)
+		{
+			for (int k = 0; k < height1; k++)
+			{
+				int a = *(matrix1 + i * max_size + k) + *(matrix2 + i * max_size + k);
+				*(result_matrix + i * max_size + k) = a;
+			}
+		}
+		my_print((int*)result_matrix, length2, height1);

```

Comment:
These functions should not produce any output. Just the arithmetical computations

Reply:
Fix it in all other